test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -8360,7 +8360,27 @@ def cleanup_jobs(self, extend=None, runas=None):
             delete_xt += 'deletehist'
             select_xt = 'x'
         job_ids = self.select(extend=select_xt)
-        if len(job_ids) > 0:
+
+        # Make sure the current user is a manager. Some tests might have
+        # unset the mangers attribute. Ignore 'Duplicate entry in the list'
+        # error if the current user was already a manager
+        current_user = pwd.getpwuid(os.getuid())[0]
+        a = {ATTR_managers: (INCR, current_user + '@*')}
+        try:
+            self.manager(MGR_CMD_SET, SERVER, a, sudo=True)
+        except PbsManagerError:
+            pass
+
+        # Turn off scheduling so jobs don't start when trying to
+        # delete. Restore the orignial scheduling state
+        # once jobs are deleted.
+        sched_state = self.status(SERVER, {'scheduling'})
```

Comment:
Just get the state ('True', 'False') here.
Something like:
`sched_state = self.status(SERVER, {'scheduling'})[0]['scheduling']`